web_widget_kanban_many2one/static/src/js/web_widget_kanban_many2one.js

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2019 Sunflower IT <www.sunflowerweb.nl>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+ */
+
+odoo.define('web_widget_kanban_many2one.kanban_editable', function(require){
+
+    var AbstractField = require('web.AbstractField');
+    var core = require('web.core');
+    var field_registry = require('web.field_registry');
+
+    var _t = core._t;
+
+    var many2one_kanban = AbstractField.extend({
+        template: _t('Kanban_editable_temp'),
+        events: _.extend({}, AbstractField.prototype.events, {
+            'click .selectM2MField': function(event){
```

Comment:
Ugly name for the selector, don't use UpperCamelCase. And it's an Many2One field not Many2Many.
Please, don't use anonymous functions like this if not are strictly necessary, better implement it in the class to easy inheritance.

Please, follow the OCA javascript style rules, see https://raw.githubusercontent.com/OCA/pylint-odoo/master/pylint_odoo/examples/.jslintrc